zerver/migrations/0371_invalid_characters_in_topics.py

```diff
@@ -49,7 +49,7 @@ def fix_topics(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None
                     continue
 
                 # We don't want empty topics for stream messages, so we
-                # use (no topic) if the above clean-up leaves us with an empty string.
+                # use no topic if the above clean-up leaves us with an empty string.
```

Comment:
This change is not required.

Reply:
Thanks for letting me know, I have now fixed this.